src/common/icons/Mobile.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function Mobile({ styling }) {
+  return (
+    <svg
+      viewBox="0 0 13 20"
+      xmlns="http://www.w3.org/2000/svg"
+      className={styling}
+    >
+      <path
+        opacity="0.5"
```

Comment:
We tend to avoid hard-coding styling in the svgs (in these cases `fill` and `opacity`). By not hard-coding these properties we can then set the styling on a case by case basis using the param `styling` which applies classes to the svg.